extras/cronwrapper

```diff
@@ -15,6 +15,27 @@
 #         (typically /var/log/daemon.log or /var/log/syslog)
 #
 
+sendSlackMessage() {
+    escapedText=$(cat ${1} | sed 's/"/\"/g' | sed "s/'/\'/g" )
+    slackUsername=${SLACK_USERNAME:-plexupdate}
+
+    if [ $RET -eq 10 ] || [ $RET -eq 0 ]; then
+        color="#36a64f"
+    else
+        color="#ad2d2d"
+    fi
+    ts=`date +%s`
+    attachment="{\"color\": \"$color\",\"footer\": \"Plex Update\",\"title\": \"Log File\", \"text\": \"$escapedText\", \"ts\": $ts}"
+
+    if [ ! -z "$SLACK_CHANNEL" ]; then
+        json="{\"channel\": \"$SLACK_CHANNEL\", \"username\": \"$slackUsername\", \"attachments\": [$attachment]}"
+    else
+        json="{\"username\": \"$slackUsername\", \"attachments\": [$attachment]}"
+    fi
+
+    curl -s -X POST -H 'Content-type: application/json' -d "$json" "$SLACK_WEBHOOK_URL"
```

Comment:
There are a lot of times I've wanted to use curl in here, but since we already have a dependency on `wget`, you should use it instead. Unless you REALLY want to get your hands dirty by completely getting rid of our dependency on wget and using curl for everything instead. :smile:

Reply:
Works for me, I'll try to incorporate wget instead of curl.